src/components/organisms/GiftRankFilters/index.tsx

```diff
@@ -0,0 +1,53 @@
+import styled from '@emotion/styled';
+import { useContext } from 'react';
+
+import { GiftActionButtons, GiftGroupButtons } from '@/components/contexts/GiftRankContext';
+import { ActionSpecificationButton } from '@/components/molecules/ActionSpecificationButton';
+import { FlexItemsBox } from '@/components/molecules/FlexItemsBox';
+import { GroupSpecificationButton } from '@/components/molecules/GroupSpecificationButton';
+import { Heading2 } from '@/components/molecules/Heading2';
+
+const hContent = `실시간 급상승 선물랭킹`;
+
+export const GiftRankFilters = () => {
+  const groupSpecificationButtons = useContext(GiftGroupButtons);
+  const actionSpecificationButtons = useContext(GiftActionButtons);
+
+  return (
+    <Wrapper>
+      <Heading2 hContent={hContent} />
+      <GroupSpecificationFilter
+        flexItems={groupSpecificationButtons}
+        FlexItemComponent={GroupSpecificationButton}
```

Comment:
넘기는 props가 컴포넌트이더라도, 소문자로 통일 + 더 일반적인 이름으로 넘기면 좀더 직관적으로 읽히지 않을까요?

```suggestion
        items={groupSpecificationButtons}
        component={GroupSpecificationButton}
```

Reply:
저는 컴포넌트를 줘야 한다는 것을 명시하기 위해 대문자로 시작했습니다.
그런데 제가 너무 복잡하게 생각했던 것 같습니다.그냥 component를 명시하니 진짜 직관적인 것 같습니다. 
그동안 이름을 부여할 때, 이름만 보고도 어떤 역할을 하는 컴포넌트인지를 명시하고 싶어 구체적인 이름을 부여했습니다. 리뷰를 보고 직관성에 가중치를 더 부여해도 좋겠다는 생각을 했습니다.
감사합니다!